joomla/joomla-cms
# [4.0] Multilanguage. Set the multilangstatus module as published or not in the modules manager depending on the status of the languagefilter plugin

4.0 automatically displays the multilangstatus module when the languagefilter plugin is enabled.

This is a nice feature, but, when you load the administrator modules manager, the multilangstatus module appears as unpublished, which is confusing. 
(Note: also, if published and position set to status it shows a weird module position in the status bar. Did not correct that) 

### Summary of Changes
Creates a new Helper to publish or not the module depending on the languagefilter setting.

### Testing Instructions
Patch, then switch the languagefilter to enabled or not, then verify that the module is published or not in the Module Manager.

### WIP : Todo
Define the exception messages. Help is welcome.
Also, I guess it should be improved to check if the module exists before proceeding => DONE
